tests/models/seamless_m4t/test_modeling_seamless_m4t.py

```diff
@@ -414,9 +414,11 @@ def _get_encoder_outputs(
         encoder_outputs["last_hidden_state"] = encoder_outputs.last_hidden_state.repeat_interleave(
             num_interleave, dim=0
         )
+        generation_config = copy.deepcopy(model.generation_config)
+        model._prepare_special_tokens(generation_config)
         input_ids = (
             torch.zeros(input_ids.shape[:2], dtype=torch.int64, layout=input_ids.layout, device=input_ids.device)
-            + model._get_decoder_start_token_id()
+            + generation_config.decoder_start_token_id
```

Comment:
do we have to do the + and not just use the decoder start toekn id?

Reply:
I think the + here makes it a tensor of all `decoder_start_token_id` (as opposed to concatenation).